### DISCUSSION TITLE
ImageMagick 6.3.5 --> 'magick/magick_libMagick_la-blob.lo' failed

### DISCUSSION THREAD

**Discussion by Javeed-JargonHandlers** [ImageMagick/ImageMagick#4204] [Help]
Hi,

I am trying to install ImageMagick 6.3.5

and I am getting an error while --> make

Makefile:8265: recipe for target 'magick/magick_libMagick_la-blob.lo' failed
make: *** [magick/magick_libMagick_la-blob.lo] Error 1

Installing on Centos 7

Please help me installing the old version.

**Comment by urban-warrior** [comment]
Why ImageMagick 6.3.5?  The latest release is 6.9.12-2.  Try the latest release and see if that builds without complaint.

**Comment by Javeed-JargonHandlers** [comment]
I am trying to use diascope --> http://diascope.sourceforge.net/

Here it says it is tested and worked on the 6.3.0 / 6.3.5 version of ImageMagick.

So I am trying with the old version. Could you please guide me in which any of the recent versions support diascope?

Thanks in advance
